core/src/main/java/com/datastax/oss/driver/internal/core/resolver/ValidResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.datastax.oss.driver.internal.core.resolver;
```

Comment:
Let's move it to `core/src/main/java/com/datastax/oss/driver/internal/core/resolver/MockResolver/`